mtest/libmscore/keysig/tst_keysig.cpp

```diff
@@ -127,15 +129,47 @@ void TestKeySig::keysig_78216()
       QVERIFY2(m1->findSegment(SegmentType::KeySig, m1->endTick()) == nullptr, "Should be no keysig at end of measure 1.");
       QVERIFY2(m2->findSegment(SegmentType::KeySig, m2->endTick()) == nullptr, "Should be no keysig at end of measure 2.");
       QVERIFY2(m3->findSegment(SegmentType::KeySig, m3->endTick()) == nullptr, "Should be no keysig at end of measure 3.");
+
+      delete score;
       }
 
+//---------------------------------------------------------
+//   concertPitch
+//---------------------------------------------------------
+
 void TestKeySig::concertPitch()
       {
       MasterScore* score = readScore(DIR + "concert-pitch.mscx");
       score->cmdConcertPitchChanged(true, true);
       QVERIFY(saveCompareScore(score, "concert-pitch-01-test.mscx", DIR + "concert-pitch-01-ref.mscx"));
       score->cmdConcertPitchChanged(false, true);
       QVERIFY(saveCompareScore(score, "concert-pitch-02-test.mscx", DIR + "concert-pitch-02-ref.mscx"));
+
+      delete score;
+      }
+
+//---------------------------------------------------------
+//   preferSharpFlat
+//---------------------------------------------------------
+
+void TestKeySig::preferSharpFlat()
+      {
+      MasterScore* score1 = readScore(DIR + "preferSharpFlat-1.mscx");
+      auto parts = score1->parts();
+      Part* part1 = parts[0];
+      part1->setPreferSharpFlat(PreferSharpFlat::FLATS);
+      score1->update();
+      score1->setLayoutAll();
```

Comment:
Why is this just not changing the key signature? Setting to `FLATS` does change the key signature for my self-built version but why not the test?

Reply:
Got it, have to call `transpositionChanged()`, it's done automatically if the property is modified inside MuseScore.